pkg/build/buildkit/errors.go

```diff
@@ -22,10 +22,35 @@ import (
 	oktetoErrors "github.com/okteto/okteto/pkg/errors"
 	"github.com/okteto/okteto/pkg/okteto"
 	"github.com/okteto/okteto/pkg/registry"
+	"google.golang.org/grpc/status"
 )
 
-// getErrorMessage returns the parsed error message
-func getErrorMessage(err error, tag string) error {
+var (
+	// ErrBuildConnecionFailed is returned when the connection to buildkit fails
+	ErrBuildConnecionFailed = errors.New("build connection failure")
+
+	grpcRetryableCodes = map[uint32]interface{}{
+		13: nil, // transport is closing (e.g. client buildkit pod is being deleted)
+		14: nil, // unavailable (e.g. no buildkit service available)
+		15: nil, // Data loss: request did not complete
+	}
+)
+
+type OktetoCommandErr struct {
```

Comment:
i am confused by the naming, does this mean is a command from the okteto manifest or is a command of buildkit running? :/ 
if is the second, should we remove okteto from the type?

Reply:
It's the name of the command inside buildkit. I'll change it if its clearer